src/SFA.DAS.AssessorService.Application/Handlers/Staff/LearnerDetailHandler.cs

```diff
@@ -52,12 +64,12 @@ public async Task<LearnerDetail> Handle(LearnerDetailRequest request, Cancellati
                 {
                     CalculateDifferences(logs);
                 }
-                
+
 
                 certificateData = JsonConvert.DeserializeObject<CertificateData>(certificate.CertificateData);
                 epao = await _organisationRepository.Get(certificate.OrganisationId);
             }
-            
+
             var learnerDetail = new LearnerDetail()
```

Comment:
Just a suggestion.  I would put this into a private mapper method to eliminate the bloat inside this function.  Your choice though.

Reply:
yep hang over from past from daves code will see if I get time to resolve for tommorrow.